src/components/signup/SmsForm.js

```diff
@@ -212,6 +222,17 @@ const getStylesFromProps = ({ theme }) => ({
     marginTop: 'auto',
     marginBottom: theme.sizes.defaultDouble,
   },
+  successIconStyle: {
+    borderWidth: 1,
+    borderRadius: '50%',
+    borderColor: theme.colors.primary,
+    position: 'relative',
+    height: getDesignRelativeHeight(48),
```

Comment:
we only use design relative height where it matters. margins, paddings, illustrations.
also notice design relative height has second parameter, if not set to false it is true by default, true being dont make it wider/heigher than design, basically it means it will only reduce the height/width on smaller devices and not make it larger on larger devices

Reply:
Done